video_xblock/static/js/studio-edit-utils.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * Auxiliary functions for studio editor modal's JS.
+ */
+
+/**
+ * Prepare data to be saved to video xblock.
+ */
+function fillValues(fields) {
+    'use strict';
+    var values = {};
+    var notSet = []; // List of field names that should be set to default values
+    var i;
+    var field;
+    for (i = 0; i < fields.length; i++) {
+        field = fields[i];
+        if (field.isSet()) {
+            values[field.name] = field.val();
+        } else {
+            notSet.push(field.name);
+        }
+        // Remove TinyMCE instances to make sure jQuery does not try to access stale instances
+        // when loading editor for another block:
+        if (field.hasEditor()) {
+            field.removeEditor();
+        }
+    }
+    return {values: values, defaults: notSet};
+}
+
+
+/**
+ * Display message with results of a performed action (e.g. a transcript manual or automatic upload).
+ */
+function showStatus(message, type, successMessageElement, errorMessageElement) {
```

Comment:
I think it can be simplified and we can use only 1 message container (MessageElement) + use type as a className to change styling. For example:
```js
function showStatus(el, type, message {
  ...
  el
    .removeClass()  // remove all previous classes
    .addClass('is-' + type) // add a new one to apply new styling. For example: 'is-success|is-error'
    .html(message);  // replaces current content with a new one.
  ...
}
```